components/Quiz.tsx

```diff
@@ -1,59 +1,56 @@
 "use client"
 
+import { FormEventHandler } from "react"
 import { Button } from "@shadcn/ui/button"
-import { Separator } from "@shadcn/ui/separator"
-import { FormEventHandler, useState } from "react"
+
 import { Typography } from "./Typography"
-import { Textarea } from "@shadcn/ui/textarea"
 
-/**
- * Don't see a need for the { isCorrect } prop now as we don't have the answer as an array
- */
-interface Props {
-  question: string
-  answer: string
+type Answer = {
+  text: string
+  isCorrect: boolean
 }
 
-function validateAnswer(answer: string) {
-  return answer.replaceAll("", "").toLowerCase()
+interface Props {
+  question: string
+  answers: Answer[]
 }
 
-export function Quiz({ question, answer }: Props) {
-  const [isCorrect, setIsCorrect] = useState(false)
-
+export function Quiz({ question, answers }: Props) {
   const handleSubmit: FormEventHandler<HTMLFormElement> = (event) => {
     event.preventDefault()
+  }
 
-    const userResponse = validateAnswer(event.currentTarget.quiz.value)
-    const correctAnswer = validateAnswer("Coursition")
+  const isMultipleCorrect = answers.filter((answer) => answer.isCorrect).length > 1
 
-    if (userResponse !== correctAnswer) return
-    setIsCorrect(true)
-  }
+  const SelectAlert = () => (
+    <>
+      {isMultipleCorrect ? (
```

Comment:
**suggestion:** We should use simpler patters, so `isMultipleCorrect && ...`

Reply:
Okay 👍🏽